src/types/Order.ts

```diff
@@ -0,0 +1,24 @@
+export enum OrderEnum {
+  asc = 'asc',
+  desc = 'desc',
+}
+
+export enum SortOrderEnum {
+  Sort = 'sort',
+  Order = 'order',
+}
+
+export enum FilterEnum {
```

Comment:
```suggestion
export enum FilterBy {
```
не потрібно писати енам, достатньо просто назви